versions.props

```diff
@@ -35,11 +35,11 @@ io.airlift:airline = 0.8
 io.atomix.catalyst:* = 1.1.2
 io.atomix.copycat:* = 1.2.0
 io.atomix:* = 1.0.0-rc9
-io.dropwizard.metrics:metrics-* = 3.2.3
-io.dropwizard.modules:* = 0.9.0-1
-io.dropwizard:* = 0.9.3
+io.dropwizard:* = 1.3.9
+io.dropwizard.metrics:metrics-* = 3.2.6
```

Comment:
Yeah I think we want to keep this because spark and hadoop are still on 3.X, and atlasdb is so widely used we don't want consumers to start thinking they need to pick up 4.X